randovania/games/prime1/json_data/Chozo Ruins.txt

```diff
@@ -2207,8 +2207,26 @@ Extra - aabb: [743.8239, -217.36417, 29.29987, 834.11273, -144.96254, 81.37303]
   * Normal Door to Reflecting Pool Access/Door to Hall of the Elders
   * Extra - dock_index: 0
   * Extra - nonstandard: False
+  > Door to Crossway Access South
+      All of the following:
+          # https://youtu.be/C5DBRQ4gCIY
+          Space Jump Boots and Knowledge (Advanced) and L-Jump (Expert) and Standable Terrain (Intermediate)
```

Comment:
Expert is an overstatement

Reply:
I had it as advanced before, but wasn't too sure if it should be there or expert, even after looking at what is already in those. Advanced is fine?

Reply:
intermediate or advanced, but definitely not expert